examples/horizontal_convection.jl

```diff
@@ -0,0 +1,386 @@
+    
+    ζ_plot = contourf(xζ, zζ, clamp.(ζ_snapshot', -ζlim, ζlim);
+                      clims=(-ζlim, ζlim), levels = ζlevels,
+                      color = :balance, kwargs...)
+    ζ_title = @sprintf("vorticity, (∂u/∂z - ∂w/∂x) √(H/b⋆) @ t=%1.2f", t)
+    
+    χ_plot = contourf(xs, zs, χ_snapshot';
+                      clims = (0, χlim), levels = χlevels,
+                      color = :dense, kwargs...)
+    χ_title = @sprintf("buoyancy dissipation, κ|∇b|² √(H/b⋆⁵) @ t=%1.2f", t)
+    
+    plot(s_plot, b_plot, ζ_plot, χ_plot,
+           size = (700, 1200),
+           link = :x,
+         layout = Plots.grid(4, 1),
+          title = [s_title b_title ζ_title χ_title])
+end
+
+mp4(anim, "horizontal_convection.mp4", fps = 16) # hide
+
+# At higher Rayleigh numbers the flow becomes much more vigorous. See, for example, an animation
+# of the voricity of the fluid at ``Ra = 10^{12}`` on [vimeo](https://vimeo.com/573730711). 
+
+# ### The Nusselt number
+#
+# Often we are interested on how much the flow enhances mixing. This is quantified by the
+# Nusselt number, which measures how much the flow enhances mixing compared if only diffusion
+# was in operation. The Nusselt number is given by
+#
+# ```math
+# Nu = \frac{\langle \chi \rangle}{\langle \chi_{\rm diff} \rangle} \, ,
+# ```
+#
+# where angle brackets above denote both a volume and time average and  ``\chi_{\rm diff}`` is
+# the buoyancy dissipation that we get without any flow, i.e., the buoyancy dissipation associated
+# with the buoyancy distribution that satisfies
+#
+# ```math
+# \kappa \nabla^2 b_{\rm diff} = 0 \, ,
+# ```
+#
+# with the same boundary conditions same as our setup. In this case we can readily find that
+#
+# ```math
+# b_{\rm diff}(x, z) = b_s(x) \frac{\cosh \left [2 \pi (H + z) / L_x \right ]}{\cosh(2 \pi H / L_x)} \, ,
+# ```
+# which implies ``\langle \chi_{\rm diff} \rangle = \kappa b_*^2 \pi \tanh(2 \pi Η /Lx)``.
+#
+# We use the loaded `FieldTimeSeries` to compute the Nusselt number from buoyancy and the volume
+# average kinetic energy of the fluid.
+#
+# First we compute the diffusive buoyancy dissipation, ``\chi_{\rm diff}`` (which is just a
+# scalar):
+
+H, Lx = file["grid/Lz"], file["grid/Lx"]
+κ = file["closure/κ/b"]
+
+χ_diff = κ * b★^2 * π * tanh(2π * H / Lx)
+nothing # hide
+
+# We then create two `ReducedField`s to store the volume integrals of the kinetic energy density
+# and the buoyancy dissipation. We need the `grid` to do so; the `grid` can be recoverd from
+# any `FieldTimeSeries`, e.g.,
+
+grid = b_timeseries.grid
+
+∫ⱽ_s² = ReducedField(Nothing, Nothing, Nothing, CPU(), grid, dims=(1, 2, 3))
+∫ⱽ_mod²_∇b = ReducedField(Nothing, Nothing, Nothing, CPU(), grid, dims=(1, 2, 3))
+
+# We recover the time from the saved `FieldTimeSeries` and construct two empty arrays to store
+# the volume-averaged kinetic energy and the instantaneous Nusselt number,
+
+t = b_timeseries.times
+
+KineticEnergy, Nu = zeros(length(t)), zeros(length(t))
```

Comment:
Technically `KineticEnergy` violates style since its an instances, not a type. I guess it should be `kinetic_energy`.

Reply:
But I'm not changing `Nu` --> `nu` :)

Reply:
perhaps I'm OK changing `Nu` -> `nusselt`

Reply:
`Nu` is ok! I guess if we had to capitalize we should write `Kinetic_Energy`. But kinetic and energy aren't names, so...